coverage-tests.js

```diff
@@ -1548,7 +1548,7 @@ test('should send enablePatterns when specified in check settings', {
 })
 
 test('should send custom batch properties', {
-  page: 'Default',
+  features: ['image'],
```

Comment:
why?

Reply:
by default it's part of selenium tests for no good reason, while the changes are in the base class, and images have much less tests (11 vs ~300).

Reply:
using the `image` feature makes no sense to me. Because this feature indicates that you can check images, this has nothing to do with this test. But I agree that this test could be used for images SDK as well. But we have to come up with another feature name. I suggest `non-visual` to indicate that this test doesn't perform any visual comparison.
```suggestion
  features: ['non-visual'],
```